commcare_connect/opportunity/tests/test_views.py

```
@@ -142,26 +142,47 @@ def test_add_budget_existing_users_for_managed_opportunity(
     indirect=True,
 )
 @pytest.mark.django_db
-def test_approve_visit(
+def test_approve_visits(
     client: Client,
     organization,
```

Ruff (0.14.8)

Unused function argument: `organization`

(ARG001)